library/Garp/Model/Db/User.php

```diff
@@ -298,6 +298,10 @@ public function invalidateEmailAddress(Garp_Db_Table_Row $user, $updateOrInsert
         $validationTokenColumn = $authVars['token_column'];
         $emailValidColumn = $authVars['email_valid_column'];
 
+        if ($updateOrInsert == 'insert' && $user[$emailValidColumn]) {
```

Comment:
Nah, dat is een beetje hetzelfde maar dan impliciet (zelfde als `return null`).  
Maar aan het einde van de functie zit `return false` ook achter `if ($user->save())`, dus als dat niet lukt lijkt me een `return false` op zijn plaats.  
Maar in jouw geval is de actie positief, dus die twee gevallen hoeven wat mij betreft niet hetzelfde terug te geven.